go/teams/chain.go

```diff
@@ -1375,6 +1390,31 @@ func (t *TeamSigChainPlayer) addInnerLink(
 		res.newState = prevState.DeepCopy()
 		t.updateInvites(&res.newState, additions, cancelations)
 		return res, nil
+	case libkb.LinkTypeSettings:
+		err = libkb.PickFirstError(
+			allowInflate(false),
+			hasPrevState(true),
+			hasName(false),
+			hasMembers(false),
+			hasParent(false),
+			hasInvites(false),
+			hasSubteam(false),
+			hasPerTeamKey(false),
+			hasSettings(true))
+		if err != nil {
+			return res, err
+		}
+
+		if prevState.IsImplicit() {
+			return res, NewImplicitTeamOperationError(payload.Body.Type)
+		}
+
```

Comment:
Only admins or implicit admins can make these links, right? Do this: https://github.com/keybase/client/blob/master/go/teams/chain.go#L863

Reply:
Done, thanks